### ISSUE TITLE
Add the InstiApp topic

### ISSUE THREAD

**Issue by pulsejet** [DevCom-IITB/instiapp-flutter#1]
Hi @tastelessjolt!

Found this a while ago, love the initiative. I have zero experience with flutter, but I'd be more than glad to help out if you need any help regarding the API docs or working.

Just a couple of points I'd like to put forward for now:
1) I'd strongly suggest against implementing any of the event add/body edit etc. functionality natively (if you're planning to), and just put the webapp in a webview as the Android app does instead. The whole thing has a really terrible and breaking-change prone API which you don't want to work with.
2) A list of API endpoints (terribly organised for now) can be found at https://api.insti.app/docs/

I could add you as a developer on the InstiApp body in case you need to make a few test events here and there. Just need to ensure notifications are turned off before creating the event and delete it when done.

Now for this particular issue, I'm trying to collect all relevant repositories at [#instiapp](https://github.com/topics/instiapp), so it'll be cool if you could add this and other relevant topics to the repo :)

**Comment by tastelessjolt** [comment]
Even I have zero experience, I'm doing this to learn Flutter in a way. I'll ask you if I have any API related questions. 
As for your first point, I concur. 

I'll ask you when I come to implementing `event`. As for now, it's in a very nascent stage, just getting comfortable with Flutter. 

And I don't have a Macintosh system with me so, I can't test it on iOS even though the readme says iOS :P. And I recently messed my Hackintosh. So, it'd help if someone could test it on iOS. Ideally, it should work. But there might be some changes needed. 

Any PRs are welcome.

As for the topics, I just added and am closing this issue.
